modules/zivid/experimental/_pixel_mapping.py

```diff
@@ -0,0 +1,50 @@
+"""Module for experimental pixel mapping. This API may change in the future."""
+
+import _zivid
+
+
+class PixelMapping:
+    """Pixel mapping from subsampled to full resolution.
+
+    Required when mapping an index in a subsampled point cloud to e.g. a full resolution 2D image.
+    """
+
+    def __init__(self, row_stride=1, col_stride=1, row_offset=0, col_offset=0) -> None:
```

Comment:
We don't generally do type hints in this repo, so you can drop the "-> None".

Reply:
https://github.com/zivid/zivid-python/pull/251/commits/1bc2593126b8ced67de2661e105a6b8e420020d7